src/main/kotlin/no/nav/familie/baks/infotrygd/feed/config/KafkaConfig.kt

```diff
@@ -0,0 +1,72 @@
+package no.nav.familie.baks.infotrygd.feed.config
+
+import org.apache.kafka.clients.CommonClientConfigs
+import org.apache.kafka.clients.consumer.ConsumerConfig
+import org.apache.kafka.common.config.SslConfigs
+import org.apache.kafka.common.serialization.StringDeserializer
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Profile
+import org.springframework.kafka.annotation.EnableKafka
+import org.springframework.kafka.config.ConcurrentKafkaListenerContainerFactory
+import org.springframework.kafka.core.ConsumerFactory
+import org.springframework.kafka.core.DefaultKafkaConsumerFactory
+import org.springframework.kafka.listener.ContainerProperties
+
+@Configuration
+@EnableKafka
+@Profile("preprod", "prod")
+class KafkaConfig(
+    @Value("\${KAFKA_BROKERS:localhost}") private val kafkaBrokers: String,
+    @Value("\${KAFKA_TRUSTSTORE_PATH}") private val kafkaTruststorePath: String,
+    @Value("\${KAFKA_CREDSTORE_PASSWORD}") private val kafkaCredstorePassword: String,
+    @Value("\${KAFKA_KEYSTORE_PATH}") private val kafkaKeystorePath: String
+) {
+
+    @Bean
+    fun consumerFactory(): ConsumerFactory<String, String> {
+        return DefaultKafkaConsumerFactory(consumerConfigs())
+    }
+
+    @Bean
+    fun concurrentKafkaListenerContainerFactory(
+        kafkaErrorHandler: KafkaErrorHandler
+    ): ConcurrentKafkaListenerContainerFactory<String, String> =
+        ConcurrentKafkaListenerContainerFactory<String, String>().apply {
+            setConcurrency(1)
+            containerProperties.ackMode = ContainerProperties.AckMode.MANUAL
+            consumerFactory = consumerFactory()
+            setCommonErrorHandler(kafkaErrorHandler)
+        }
+
+    fun consumerConfigs() = mapOf(
+        ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG to kafkaBrokers,
+        ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG to StringDeserializer::class.java,
+        ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG to StringDeserializer::class.java,
+        ConsumerConfig.GROUP_ID_CONFIG to "familie-tilbake",
+        ConsumerConfig.CLIENT_ID_CONFIG to "consumer-familie-tilbake-1",
+        ConsumerConfig.AUTO_OFFSET_RESET_CONFIG to "latest",
```

Comment:
latest gir muligheten til at man mister meldingen. earliest med idempotent tjeneste er bedre

Reply:
ser på denne hvor stor endringer er. Hvis det er for stor, kan det gjøres senere